styles.scss

```diff
@@ -1,12 +1,208 @@
 /* Ad styles */
 
-[data-ethical-ad-publisher] {
-  background: #f0f0e0;
-  font-family: monospace;
-  margin: 1rem;
-  padding: 1rem;
+// Sizes
+$width-image-horizontal: 320px;
```

Comment:
All the existing ads in our system are 240x180 and displayed at 120x90.

Reply:
This is not the image width, this is the bounding box width